infotiles/app.js

```diff
@@ -13,17 +13,20 @@ lbs.apploader.register('infotiles', function () {
 
         appConfig = appConfig || {};
         this.showOnEmpty = appConfig.showOnEmpty == null && true || appConfig.showOnEmpty;
-/*
```

Comment:
This change will bring back the timer functionality to life. It was previously commented out. Should the infotiles really have a setting to autorefresh themselves, @TommyLindh2 ? Seems dangerous for the cloud environment.

Reply:
I got some more information from Richard Rehnstam. Apparently there was more code that could be removed. That should be done before we merge this into master.